operator/api/redpanda/v1alpha3/nodepool_types.go

```diff
@@ -96,3 +98,9 @@ type BrokerTemplate struct {
 	// rack: Expr(node_annotation('k8s.io/failure-domain')),
 	// TODO deprecate and move me into CEL functions.
 }
+
+type BrokerImages struct {
+	Redpanda      string `json:"redpanda"`
```

Comment:
Harkening back to the conversations about webhooks and defaulting. This makes it a bit difficult to specify a pull through proxy and not otherwise touch the images.

The Sidecar is probably the most important example because the operator injects it's current tag into this value. If someone wants to just override the repository, how would they go about doing so?

Personally, I really dislike the chart's method of splitting up the image (which also makes it difficult / impossible to use sha256 manifests) but that leaves us with templating strings if we're going to make this just a string.